sdk/core/core-client/package.json

```diff
@@ -19,37 +19,29 @@
-    "build:browser": "npm run build:ts && cross-env ONLY_BROWSER=true rollup -c 2>&1",
-    "build:node": "npm run build:ts && cross-env ONLY_NODE=true rollup -c 2>&1",
-    "build:samples": "echo Skipped.",
-    "build:test": "npm run build:ts && npm run bundle:test",
-    "build:test:browser": "npm run build:ts && npm run bundle:test:browser",
-    "build:test:node": "npm run build:ts && npm run bundle:test:node",
-    "build:ts": "tsc -p .",
+    "build:samples": "echo Obsolete",
+    "build:test": "tsc -p . && rollup -c 2>&1",
     "build:types": "downlevel-dts types/latest/ types/3.1/",
-    "build": "npm run build:ts && rollup -c 2>&1 && api-extractor run --local && npm run build:types",
-    "bundle:test": "rollup -c rollup.test.config.js 2>&1",
-    "bundle:test:browser": "cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
-    "bundle:test:node": "cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
-    "check-format": "prettier --list-different \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "clean": "rimraf dist dist-* types *.tgz *.log",
+    "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local && npm run build:types",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* temp types *.tgz *.log",
+    "docs": "typedoc --excludePrivate --excludeNotExported --excludeExternals --stripInternal --mode file --out ./dist/docs ./src",
     "execute:samples": "echo skipped",
-    "extract-api": "npm run build:ts && api-extractor run --local",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "echo skipped",
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
     "lint": "eslint package.json api-extractor.json src test --ext .ts",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
-    "test:browser": "npm run build:test:browser && npm run unit-test:browser && npm run integration-test:browser",
-    "test:node": "npm run build:test:node && npm run unit-test:node && npm run integration-test:node",
-    "test": "npm run clean && npm run build:ts && npm run bundle:test:node && npm run unit-test:node && npm run bundle:test:browser && npm run unit-test:browser && npm run integration-test:node && npm run integration-test:browser",
```

Comment:
Standardization is good, though one feature of the above that I *really* like is the minimal amount of work to execute the tests quickly in node and pass/fail before moving on to the browser. 

The new test command spends a lot of time packing the tests for browser when they maybe don't even pass in node, which slows down your debug loop considerably.

Perhaps we could tweak our default command to not spend so much time in rollup before executing the node tests for mocha? Since we don't need rollup for node tests, it seems like it should be possible

Reply:
I agree! I ditched bundling before running node tests.

Reply:
I also added running the clean script before building tests.